### DISCUSSION TITLE
Does anyone know how I can fix this error?

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2704] [Q&A]
Hello everyone,

I have converted my Abaqus input file to OpenRadioss input file format. I have some issues with the converted file. I face with an error and I don't know how I can fix it. The error is as below:
ERROR ID : 100101

** ERROR IN INPUT FORMAT

DESCRIPTION :  

-- BLOCK: /NODE

-- LINE: 		179				.015318			   -.09959			  -.119714	

STRING "179." found instead of an integer

SOLUTION :     

Please check input format and fields description defined in the Reference Guide.
-----------------------------------------------------------------------------------------------------
ERROR ID : 100101

** ERROR IN INPUT FORMAT

DESCRIPTION :  

-- BLOCK: /SHELL/7

-- LINE: /SHELL/7	

STRING "	   " found instead of an integer

SOLUTION :     

Please check input format and fields description defined in the Reference Guide.
-----------------------------------------------------------------------------------------------------

Does anyone know what the problem is. I have attached the input file to this discussion. 

Thank you!
[Powder_OpenRadioss.zip](https://github.com/OpenRadioss/OpenRadioss/files/15478326/Powder_OpenRadioss.zip)

**Comment by MarianBulla** [comment]
Hi @Majid-MHZ,
thank you for your interest in OpenRadioss.
This is an easy issue, to solve :-)
The input format is strictly defined. 
Please do NOT use special charactes, like tabs-stops. Use 8 blancs, instead.

Please review the manual for the inputdec format. It describes in detail, how a Radioss and OpenRadioss input dec is designed:

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/ffcacf28-5297-4a05-88a0-821691a7a483)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/29eea026-7b27-4797-a389-3117101146bd)

In your case, just re-format your lines accordingly and it should work without the errors :-)

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

Thank you for your help. I'm having trouble understanding how to modify my input file. Can you please clarify how I should use the template? Do I need to create a table in Word and input the parameters into it?

Best regards,
Majid

**Reply by MarianBulla** [reply]
Dear @Majid-MHZ,
the input file is just an ASCII file.

Please have a look at some of the example maodel: https://openradioss.org/models/


Here is an example of the BumperBeam case:

#RADIOSS STARTER
#--------------------------------------------------------------------------------------------------|
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/BEGIN
test_2013_09_11                                                                 
      2022         0
                  kg                  mm                  ms
                  kg                  mm                  ms
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
#-  1. CONTROL CARDS:
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/TITLE
TEST-Run
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
#-  2. MATERIALS:
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/MAT/PLAS_JOHNS/1
generic DP600
#              RHO_I
             7.85E-6                   0
#                  E                  Nu
              210.00                  .3
#                  a                   b                   n             EPS_max            SIG_max0
                  .3                  .3                  .3                   0                   0
#                  c           EPS_DOT_0       ICC   Fsmooth               F_cut
                   0                   0         0         0                   0
#                  m              T_melt              rhoC_p                 T_r
                   0                   0                   0                   0
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
#-  3. NODES:
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/NODE
        49   -39.9775039950075    -9.9864363242814                  75
        50   -39.9264408058559   -19.9713116099398                  75
        51   -39.8470943075076   -29.9546263012189                  75
        52   -39.7397483749249    -39.936380842362                  75
        53   -39.6046868830701   -49.9165756776141                  75
        54   -39.4421937069056   -59.8952112512178                  75
        55   -39.2525527213936   -69.8722880074176                  75
        56   -39.0360478014966   -79.8478063904566                  75
        57   -38.7929628221768   -89.8217668445795                  75
        58   -38.5235816583965   -99.7941698140292                  75
.
.
.


Hope it helps you ?

Thanks and best regards,
Marian

**Reply by MarianBulla** [reply]
If you open your model, using a simple text editor, you should see something like this:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/9e86f208-73fc-4934-b4f1-aad84ebb0073)

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

I have opened my input file in a simple notepad. Despite arranging the data before, the characters and numbers are misaligned and not organized in the simple notepad. I want to know if I have to create an input file and array those numbers in their columns manually.
Isn't there a prepared template?

Best regards,
Majid

**Reply by MarianBulla** [reply]
Dear @Majid-MHZ,
it depends on your texteditor, you are using.
For some text-editors there maybe are templates for Radioss available, which you can find on the internet: 
For VS Code, there is definitely a plugin providing such functionality for Radioss (and OpenRadioss).

Just search for it.
Hope it helps you ?
Thanks and best regards,
Marian

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

I am really sorry for taking your time.
I have rectified the input file and arrange those numbers. I ran it, but I got the same errors as before.
Why is it so? I am really confused. Please have look at the attached picture.

![image](https://github.com/OpenRadioss/OpenRadioss/assets/169218266/abc95ecb-c241-4a23-b4ff-560fdad867bc)

Best regards,
Majid

**Reply by MarianBulla** [reply]
Can you add your new model here, please ?
Hard to judge without beeing able to see your model.

**Reply by SyazwanSamad** [reply]
> I am really sorry for taking your time. I have rectified the input file and arrange those numbers. I ran it, but I got the same errors as before. Why is it so? I am really confused. Please have look at the attached picture.
> 
> ![image](https://private-user-images.githubusercontent.com/169218266/334761489-abc95ecb-c241-4a23-b4ff-560fdad867bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5OTQxMTcsIm5iZiI6MTcxNjk5MzgxNywicGF0aCI6Ii8xNjkyMTgyNjYvMzM0NzYxNDg5LWFiYzk1ZWNiLWMyNDEtNGEyMy1iNGZmLTU2MGZkYWQ4NjdiYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQxNDQzMzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNGZmNzBiZDY2NDU4ZTI0YzFlODJjYTkwZDlkY2E1ODI4MzJhMzE4NWI2NWE1Mjc5NTRlOWMzOGIzZjdlNDY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Lpo-Zx2-BbnVWhYypsh3EOHPBG9IGQos5_RGkFrfnnQ)
> 
> Best regards, Majid

Seems that Xc column is still wrong. The last character should be aligned to "|" (between 3rd and 4th column)

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

Sorry for the late reply. I have uploaded the file here.

Best regards,
Majid

[Powder_Compaction_0000.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/15486531/Powder_Compaction_0000.rad.txt)

**Reply by MarianBulla** [reply]
Dear @Majid-MHZ,
please pay attention:
Integer values have only 10 columns
floating point numbers have 20 columns

In addition to that:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/ad5a7385-2447-4b33-a29c-4c9b282774e0)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/7b9684af-fb56-4c8b-98a3-596a67619228)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/53fd776c-ad5a-4fa2-9500-ae1c68f03980)

Please recheck all values and its format and aligments.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 
I hope you are doing well.

I appreciate your help. I could manage all those errors, but there are two other errors. 
I am really grateful if you help me to fix these errors. I modified the input file several times but couldn't fix the errors.
I have attached my new input file here. 

Thank you so much for your consideration.

Best regards,
Majid
![image](https://github.com/OpenRadioss/OpenRadioss/assets/169218266/b83e4202-1760-4875-a408-27366cd223c7)
[Powder_OpenRadioss.zip](https://github.com/OpenRadioss/OpenRadioss/files/15494946/Powder_OpenRadioss.zip)

**Reply by MarianBulla** [reply]
Dear @Majid-MHZ, 
You're improving :-)

But your inputdec still contains special-characters, as the ERROR ID is mentioning it.
You can make it visible with any text-editor, like TextPad or notepad++.

Please remove all tab-stops by corresponding number of blancs (just simple "spaces"):

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/a54d8e17-891b-4530-8f4c-ef4f56775f38)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/e7e8ff71-7478-43fb-a6a8-9e1b52add5d4)

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by Majid-MHZ** [comment]
Dear @MarianBulla 

I really appreciate you for your help and support.

I have modified the input file, as you mentioned. I think I have removed all the special characters and the tap-stops. 
May I request you to check the input file for the last time to see if it has any problems? 

Thank you in advance for your kindness and help.

Best regards,
Majid
[Powder_Compaction_0000.zip](https://github.com/OpenRadioss/OpenRadioss/files/15501761/Powder_Compaction_0000.zip)

**Reply by MarianBulla** [reply]
Dear @Majid-MHZ,

same issue, here:

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/64351be2-1157-43e4-84fa-1f5f5b17d78d)

Best regards,
Marian
